app/models/user.rb

```diff
@@ -6,4 +6,6 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
   include DeviseTokenAuth::Concerns::User
+
+  has_many :rents, dependent: :destroy
```

Comment:
Open questions:
1. Why did you choose `:destroy` if the client wants to have an historic of rent even if the user was deleted? 
2. What other option have you?

Reply:
Since the documentation doesn't specify what to do with the associated records when the record is destroyed, I just chose to destroy them, but for example if the client wants to have a historic of the rents, `:nullify` could've been uses instead.

Other options are:

- delete_all
- nullify
- restrict_with_exception
- restrict_with_error